src/backend/joanie/settings.py

```diff
@@ -239,19 +241,39 @@ class Base(Configuration):
 
     # Joanie settings
     JOANIE_CURRENCY = values.Value(
-        ("EUR", "\N{euro sign}"), environ_name="JOANIE_CURRENCY"
+        ("EUR", "\N{euro sign}"), environ_name="JOANIE_CURRENCY", environ_prefix=None
     )
-    JOANIE_VAT = values.Value(20, environ_name="JOANIE_VAT")
+    JOANIE_VAT = values.Value(20, environ_name="JOANIE_VAT", environ_prefix=None)
     JOANIE_INVOICE_COMPANY_CONTEXT = values.Value(
-        """
-        10 rue Stine, 75001 Paris
+        """10 rue Stine, 75001 Paris
         RCS Paris XXX XXX XXX - SIRET XXX XXX XXX XXXXX - APE XXXXX
         VAT Number XXXXXXXXX""",
         environ_name="JOANIE_INVOICE_COMPANY_CONTEXT",
+        environ_prefix=None,
     )
 
     AUTH_USER_MODEL = "core.User"
 
+    # Payment
+    PAYMENT_BACKEND = {
+        "backend": values.Value(
+            "joanie.payment.backends.payplug.PayplugBackend",
+            environ_name="PAYMENT_BACKEND",
+            environ_prefix=None,
+        ),
+        "configuration": {
+            "api_version": values.Value(
+                "2019-08-06", environ_name="PAYPLUG_API_VERSION", environ_prefix=None
+            ),
+            "secret_key": values.Value(
+                "", environ_name="PAYPLUG_SECRET_KEY", environ_prefix=None
+            ),
+            "public_key": values.Value(
+                "", environ_name="PAYPLUG_PUBLIC_KEY", environ_prefix=None
+            ),
+        },
```

Comment:
since this is linked to payplug, it would make sense to hardcode the backend name above :man_shrugging:
Someone looking for another backend would anyway override the whole setting?